task_spec/spec/acceptance/init_spec.rb

```diff
@@ -202,30 +191,46 @@ def log_output_errors(result)
       expect(output).to match(%r{ensure\s+=> 'stopped'})
     end
 
-    # Succesfully upgrade from puppet6 to puppet7
-    results = run_task('puppet_agent::install', 'target', { 'collection' => puppet_7_collection, 'version' => 'latest' })
+    # Succesfully upgrade from puppet7 to puppet8
+    results = run_task('puppet_agent::install', 'target', { 'collection' => puppet_8_collection, 'version' => 'latest' })
 
     results.each do |result|
-      logger.info("Upgraded puppet-agent to puppet7 on #{result['target']}: #{result['status']}")
+      logger.info("Upgraded puppet-agent to puppet8 on #{result['target']}: #{result['status']}")
       log_output_errors(result)
     end
 
     expect(results).to all(include('status' => 'success'))
 
     # Verify that it upgraded
     installed_version = nil
-    results = run_task('puppet_agent::version', 'target', {})
+    # With prerelease puppet8, the puppet_agent::version task returns the wrong
+    # output. To temporarily work around this, we'll run puppet --version.
+    # Revert this change once Puppet 8.0.0 has been released.
+    # results = run_task('puppet_agent::version', 'target', {})
```

Comment:
maybe file a ticket so we don't forget?

Reply:
Yes, I'm keeping a running list of things we'll need to update after the final 8.0.0 release, I'll open a PA ticket about that today